packages/sources/web-components/src/components.d.ts

```diff
@@ -6,6 +6,23 @@
  */
 import { HTMLStencilElement, JSXBase } from "@stencil/core/internal";
 export namespace Components {
+    interface VtmnBadge {
+        /**
+          * The size of the badge.
+          * @default null
+         */
+        "size": 'small' | 'medium';
```

Comment:
issue: be careful, your description do not match the type
you ts type imply that it can not be null
if you want it to be pottentially undefined add a ? 
```
size?: 'small' | 'medium'
```
so size could be either small, medium or undefined

Reply:
Thanks for your feedback :)